v2/bigtable-common/src/main/java/com/google/cloud/teleport/v2/bigtable/options/BigtableCommonOptions.java

```diff
@@ -205,9 +204,7 @@ interface ReadChangeStreamOptions extends BigtableCommonOptions.ReadOptions {
         optional = true,
         description = "Cloud Bigtable change streams metadata table ID",
         helpText =
-            "The Cloud Bigtable change streams connector metadata table ID to use. If not "
-                + "provided, a Cloud Bigtable change streams connector metadata table will automatically be "
-                + "created during the pipeline flow.")
+            "The Bigtable change streams metadata table ID.")
```

Comment:
I think we should leave the original description for this one, looks like the GCP docs never got updated